### PR TITLE
Bundle the package icon with the nupkg file.

### PR DESCRIPTION
**Pull request by teo-tsirpanis**
And some other minor tweaks.

### CODE REVIEW

**Comment by igitur** [comment]
I think this can be closed. Fixed in #229

**Comment by xoofx** [comment]
@teo-tsirpanis can you rebase of master? (which includes the fix for .NET 3.5)

**Comment by teo-tsirpanis** [comment]
@xoofx done.

File: src/Scriban/Scriban.props [hunk 1]
```diff
@@ -28,6 +28,7 @@
   </PropertyGroup>
   <ItemGroup Condition=" '$(TargetFramework)' == 'net35'">
     <Reference Include="System.Web" />
+    <PackageReference Include="jnm2.ReferenceAssemblies.net35" Version="1.0.1" />
```
Comment (xoofx) [inline src/Scriban/Scriban.props:31]:
```suggestion
```

**Comment by xoofx** [comment]
Thanks!

**Pull request merged**